### PR TITLE
Fixes mongodb kinetic

### LINKED ISSUE #233: Mongodb interface was changed in kinetic
There is a mongodb plugin here:
https://github.com/ros-planning/warehouse_ros_mongo

However we need to follow the steps detailed in the Readme to build it: https://github.com/ros-planning/warehouse_ros_mongo/blob/jade-devel/README.md

There are plans to replace mongodb for another database but nothing ready yet:
https://github.com/ros-planning/moveit/issues/123

I am not sure if we should wait until something else is realease?

### PR DESCRIPTION
**Pull request by beatrizleon**
merge after #230 

Solves  #233, still specific build instructions are needed:
https://github.com/ros-planning/warehouse_ros_mongo/blob/jade-devel/README.md

### CODE REVIEW

**Comment by toliver** [comment]
So let's wait until jenkins build passes to merge this one.

File: repository.rosinstall [hunk 1]
```diff
@@ -48,3 +48,18 @@
...
+    local-name: warehouse_ros_mongo
+    uri: https://github.com/ros-planning/warehouse_ros_mongo.git
+    version: jade-devel
+
+- git:
+    local-name: warehouse_ros
+    uri: https://github.com/ros-planning/warehouse_ros.git
+    version: jade-devel
+
+- git:
```
Comment (toliver) [inline repository.rosinstall]: This is not a ROS package, so it should be installed through some other mechanism. (e.g. ansible role)

**Review by toliver — CHANGES_REQUESTED** [review]

File: repository.rosinstall (hunk 1 above)
Comment (beatrizleon) [reply repository.rosinstall]: ok, deleted

File: sr_moveit_hand_config/package.xml [hunk 2]
```diff
@@ -32,8 +32,9 @@
   <run_depend>trac_ik_kinematics_plugin</run_depend>
   <build_depend>trac_ik_lib</build_depend>
   <run_depend>trac_ik_lib</run_depend>
-
-
+  <build_depend>mongodb</build_depend>
+  <run_depend>mongodb</run_depend>
+  <build_depend>scons</build_depend>
```
Comment (toliver) [inline sr_moveit_hand_config/package.xml]: Do this package really depend on scons, or is this a trick to facilitate the installation of mongodb?
If so, please add a comment in the package.xml

**Review by toliver — CHANGES_REQUESTED** [review]
If it partially solves the issue, can you comment on what is missing?

File: sr_moveit_hand_config/package.xml (hunk 2 above)
Comment (beatrizleon) [reply sr_moveit_hand_config/package.xml]: Yes, it is only needed for installing the mongo-cxx-driver. As we are not longer installing it here, I am going to remove the dependency

**Comment by beatrizleon** [comment]
I updated pr description and add a link to instructions to build it

**Pull request merged**